### DISCUSSION TITLE
Remark42 iframe height not resizing consistently

### DISCUSSION THREAD

**Discussion by markmonroy** [umputun/remark42#1483] [Q&A]
Posting this in discussions since it may be something to do with my Hugo theme or my browser rather than remark42. It's also a very minor nuisance.

I notice that, on a page where there aren't any comments yet, and I'm not signed in, maybe 1 out of 4 times clicking on "Sign In" will not resize the iframe, resulting in the UI being cut off like so:

![Screen Shot 2022-09-11 at 12 15 13 PM](https://user-images.githubusercontent.com/32033503/189538192-7a1d7428-e00c-4285-a304-76003ce84513.jpg)

This is in Safari on macOS and iOS as well as Chrome on macOS. Anything I can do on my end, or within the Hugo template, to make iframe resizing more responsive?

#### Editing to add:

My Hugo theme, [hugo-theme-stack](https://github.com/CaiJimmy/hugo-theme-stack), comes with a pre-built remark42 template. The javascript does appear to have some significant differences from what's in the remark42 docs, but I don't know enough javascript to know if that could be affecting the iframe height responsiveness. However I might try adjusting the template a bit to see if I can make it follow the docs more closely, and see what happens.

What my Hugo theme is generating:

```
<script>
    var remark_config = {
        host: "https:\/\/blog.example.com\/remark42",
        site_id: 'mysite',
        components: ['embed'],
        url: "https:\/\/blog.example.com\/example-post\/",
        max_shown_comments:  15 ,
        theme: document.documentElement.dataset.scheme,
        page_title: 'Example Post',
        locale: 'en',
        show_email_subscription:  false ,
        show_rss_subscription:  false ,
        simple_view:  true 
    };

    (function (c) {
        for (var i = 0; i < c.length; i++) {
            var d = document, s = d.createElement('script');
            s.src = remark_config.host + '/web/' + c[i] + '.js';
            s.defer = true;
            (d.head || d.body).appendChild(s);
        }
    })(remark_config.components || ['embed']);

    window.addEventListener('onColorSchemeChange', (e) => {
        window.REMARK42.changeTheme(e.detail);
    })
</script>
```

**Comment by paskal** [comment]
I have a similar problem on [that page](https://terrty.net/2022/self-hosted-comments-remark42/) in Safari, auth form is hidden behind other elements:
<img width="1264" alt="image" src="https://user-images.githubusercontent.com/712534/189770131-758464e2-6580-4451-a8c8-196f337602ec.png">
@akellbl4 could you please check these two errors? I don't think outdated js code is relevant at all but I'll update it in https://github.com/CaiJimmy/hugo-theme-stack/pull/683 anyway.

**Reply by akellbl4** [reply]
I see the problem. Something is triggering re-render and shrink size of the iframe. I'll take a look

**Comment by markmonroy** [comment]
Alright, that's it, I kicked in for the Patreon. Honestly, comments on my extremely low-traffic blog aren't THAT important to me, but this is the most responsive open-source project I've engaged with thus far.  I really appreciate it.

**Reply by markmonroy** [reply]
also I did adjust the javascript to match the current docs but the behavior is the same.

**Reply by akellbl4** [reply]
It's a bug, no worries. I'll fix it
